toolset/setup/linux/languages/php.sh

```diff
@@ -49,6 +49,8 @@ printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f redis
 # yaf.so
 printf "\n" | $IROOT/php-${VERSION}/bin/pecl -q install -f yaf
```

Comment:
I know you didn't add this, but isn't `yaf` a framework for PHP? Why is it installed in the base `php.sh`?